rX/snapshot-generation/nl-core-NursingIntervention-output.json

```diff
@@ -4100,10 +4081,11 @@
       "id" : "CarePlan.activity.detail.performer:healthcareProvider",
       "path" : "CarePlan.activity.detail.performer",
       "sliceName" : "healthcareProvider",
-      "short" : "A reference from one resource to another",
-      "definition" : "A reference from one resource to another.",
-      "comment" : "Each occurrence of the zib HealthProfessional is normally represented by _two_ FHIR resources: a PractitionerRole resource (instance of [nl-core-HealthProfessional-PractitionerRole](http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole)) and a Practitioner resource (instance of [nl-core-HealthProfessional-Practitioner](http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-Practitioner)). The Practitioner resource is referenced from the PractitionerRole instance. For this reason, sending systems should fill the reference to the PractitionerRole instance here, and not the Practitioner resource. Receiving systems can then retrieve the reference to the Practitioner resource from that PractitionerRole instance.\r\n\r\nIn rare circumstances, there is only a Practitioner instance, in which case it is that instance which will be referenced here. However, since this should be the exception, the nl-core-HealthProfessional-Practitioner profile is not explicitly mentioned as a target profile.",
+      "short" : "HealthcareProvider",
+      "definition" : "The health professional carrying out the nursing procedure.",
+      "comment" : "A performer MAY also be a participant in the care plan.",
```

Comment:
Like above:

.comment now reverts to the base definition from FHIR. This is not the desired behavior, it should be the value from the data type profile used here.

On the other hand, .short, .definition and .alias now use the correct values (they come from the base profile and are not overridden by anything).